src/cc_helpers.cc

```diff
@@ -263,6 +274,35 @@ bool certifier::framework::cc_trust_data::initialize_keystone_enclave_data(const
 #endif
 }
 
+bool certifier::framework::cc_trust_data::initialize_cca_enclave_data(
+                const string& attest_key_file_name,
+                const string& measurement_file_name,
+                const string& attest_endorsement_file_name) {
+
+#ifdef CCA_CERTIFIER
+    if (!cc_policy_info_initialized_) {
+      printf("%s(): Policy key must be initialized first\n", __func__);
+      return false;
+    }
+
+    if (enclave_type_ != "cca-enclave") {
+      printf("%s(): '%s' is not a simulated enclave\n", __func__, enclave_type_.c_str());
+      return false;
+    }
+    // TODO
+    byte der_cert[100];
+    if (!cca_Init(0, der_cert)) {
```

Comment:
@bitboom @jinbpark - Please advise if this has to be corrected. For now, this is just some code copied from other such implementations.

Reply:
It's correct. `cca_Init` is dummy function returned `true` always for now.